### DISCUSSION TITLE
自研APP能不能嵌入taro 小程序

### DISCUSSION THREAD

**Discussion by fangjie123** [NervJS/taro#17008] [Q & A]
Taro能不能开发小程序 集成在自己的的APP里面，在安卓、鸿蒙、iOS 上都能用

**Comment by luojing1336** [comment]
> Taro能不能开发小程序 集成在自己的的APP里面，在安卓、鸿蒙、iOS 上都能用

同样有这样的需求，并不想用uniapp的uni-sdk，想给android、ios和鸿蒙原生app里嵌入鸿蒙的小程序容器，可以打开taro小程序

**Comment by z0ffy** [comment]
你只需要一个小程序容器就可以。比如可以使用 [finclip](https://www.finclip.com) 只要 APP 内接入小程序容器之后。Taro 编译成为微信小程序即可。可玩性很高，无坑。

**Reply by luojing1336** [reply]
FinClip之前他们商务有来公司营销过，做个小调研，印象中FinClip要收费。它们的容器是开源的？

**Reply by z0ffy** [reply]
不开源要收费，基础版一年好像小几千也够用。虽说有了容器小程序就能跑起来，但是配套的整个小程序管理系统也很重要。版本控制，灰度发布啥的

**Reply by luojing1336** [reply]
> 不开源要收费，基础版一年好像小几千也够用。虽说有了容器小程序就能跑起来，但是配套的整个小程序管理系统也很重要。版本控制，灰度发布啥的



> 不开源要收费，基础版一年好像小几千也够用。虽说有了容器小程序就能跑起来，但是配套的整个小程序管理系统也很重要。版本控制，灰度发布啥的

收费就算了吧，还不如选择嵌入uniSDK
